src/allEqual.js

```diff
@@ -0,0 +1,23 @@
+import { uniq, length, lte, __, pipe } from 'ramda';
+
+/* eslint-disable */
+/**
+  * Checks that all elemnets in a list are the same
+  *
+  *
+  *
+  * @func allEqual
+  * @memberOf RA
+  * @category List
+  * @sig [Iterable a] -> boolean
+  * @params {Iterable.<*>}
+
+  * @example
+  *
+  * allEqual([ 1, 2, 3, 4 ]); //=> false
+  * allEqual([ 1, 1, 1, 1 ]); //=> true
+  */
+/* eslint-enable */
+const allEqual = pipe(uniq, length, lte(__, 1));
```

Comment:
Prefer
```js
const allEqual = curry((x, arr) => (
  all(equals(x), arr)
));
```

Reply:
It's not the same thing, this implementation had arity 2 and should be named `allEqualTo`

Reply:
I agree with @guillaumearm. I think this is a separate function: `allEqualTo`.

Reply:
Agree. I missed the previous comments. Removing this comment.